internal/pkg/template/build.go

```diff
@@ -0,0 +1,64 @@
+package template
```

Comment:
You're going to have problems using this package name with the go text/template as the names will conflict.  This will resolve to `template.Build()` and text/template to `template.Parse()` after importing. 

I would suggest keeping this inside pct as `pct/build.go`, this way all business logic methods regarding PCTs are in one package.